scripts/base/frameworks/analyzer/main.zeek

```diff
@@ -46,7 +46,7 @@ export {
 	## ports: The set of well-known ports to associate with the analyzer.
 	##
 	## Returns: True if the ports were successfully registered.
-	global register_for_ports: function(tag: Analyzer::Tag, ports: set[port]) : bool;
+	global register_for_ports: function(tag: AllAnalyzers::Tag, ports: set[port]) : bool;
```

Comment:
Also, a test checking typing of the tag types (individual vs aggregate) would good.

Reply:
I reverted this change in https://github.com/zeek/zeek/pull/1838/commits/0aafa1779498ce039213b5a655fb8589c2c00ce5, since I added new methods for the packet analyzers.